lib/core/network/api/endpoints.dart

```diff
@@ -1,21 +1,49 @@
+import 'dart:convert';
+
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+
+enum Networks { telos, telosTestnet, eos, eosTestnet }
+
 class Endpoints {
   Endpoints._();
 
-  // base url - the load balancer takes care of available features on a given node, such as history, etc
-  static const String baseUrl = 'http://mainnet.telos.net';
+  static Map<String, dynamic> getNetworkConfig({String? network}) {
+    final remoteConfig = FirebaseRemoteConfig.instance;
+    network = network ?? Networks.telos.name;
+    final conf = json.decode(remoteConfig.getValue('networks').asString());
```

Comment:
this stuff is wonky. 

I think we can create a service that handles all the retrieving of the remote configs. 

it could have functions like 
getNetworks
getX
getY

and all the implementation how to set it up and get it is encapsulated inside that service.

Reply:
That's basically what these helper methods are doing but yes, would be more consistent I think I know what you mean....